### ISSUE TITLE
Can't understand how to use it when no target temperature can be set

### ISSUE THREAD

**Issue by sycorax** [tetele/hvac_group#80]
### System Health details

![image](https://github.com/tetele/hvac_group/assets/790493/a5f03933-986c-4196-93c8-32ce5e260fa3)

I have a ViCare thermostat

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/tetele/hvac_group/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

Can't understand how to use it when no target temperature can be set

### Reproduction steps

1. Install Hvac group from HACS
2. Add Helper with vicare climate, set a temperature sensor
3. Can't set target temperature

### Debug logs

```text
2024-02-15 14:20:11.803 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration xiaomi_cloud_map_extractor which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-15 14:20:11.807 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration hvac_group which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-15 14:20:11.808 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration miele which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-15 14:20:11.810 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration worlds_air_quality_index which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-15 14:20:11.812 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-15 14:20:29.734 WARNING (MainThread) [homeassistant.components.camera] SUPPORT_ON_OFF was used from xiaomi_cloud_map_extractor, this is a deprecated constant which will be removed in HA Core 2025.1. Use CameraEntityFeature.ON_OFF instead, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues
2024-02-15 14:20:29.743 WARNING (MainThread) [homeassistant.components.camera] SUPPORT_ON_OFF was used from xiaomi_cloud_map_extractor, this is a deprecated constant which will be removed in HA Core 2025.1. Use CameraEntityFeature.ON_OFF instead, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues
2024-02-15 14:20:42.632 WARNING (MainThread) [homeassistant.helpers.entity] Entity camera.xiaomi_cloud_map_extractor (<class 'custom_components.xiaomi_cloud_map_extractor.camera.VacuumCamera'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <CameraEntityFeature.ON_OFF: 1>, please create a bug report at https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation
2024-02-15 14:20:52.647 WARNING (MainThread) [homeassistant.helpers.entity] Update of camera.xiaomi_cloud_map_extractor is taking over 10 seconds
2024-02-15 14:23:10.883 WARNING (MainThread) [homeassistant.components.climate] Entity None (<class 'custom_components.hvac_group.climate.HvacGroupClimateEntity'>) implements HVACMode(s): off, heat and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/tetele/hvac_group/issues
2024-02-15 14:23:51.582 WARNING (MainThread) [custom_components.hvac_group] No previously saved temperature, setting to 7, 35
2024-02-15 14:25:37.784 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-02-15 14:25:40.779 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry aaa for climate
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 444, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 220, in async_setup_entry
    return await component.async_setup_entry(entry)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 183, in async_setup_entry
    raise ValueError("Config entry has already been setup!")
ValueError: Config entry has already been setup!
2024-02-15 14:25:57.913 WARNING (MainThread) [homeassistant.components.camera] Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
2024-02-15 14:25:57.994 WARNING (MainThread) [homeassistant.helpers.entity] Update of camera.xiaomi_cloud_map_extractor is taking over 10 seconds
```

**Comment by tetele** [comment]
What version are you using? There are a few betas for 0.2 which address a lot of issues. I'll push 0.2.0 as a stable version today

**Comment by sycorax** [comment]
0.1.1 Thanks

**Comment by tetele** [comment]
I've published v0.2.0

**Comment by Ra72xx** [comment]
I installed this integrarion for the first time yesterday and I also encounter this error. Is this supposed to be fixed?
